week03/assignment03.go

```diff
@@ -0,0 +1,127 @@
+package week03
+
+import (
+	"fmt"
+	"math"
+	"math/rand"
+	"time"
+)
+
+const (
+	min  = 1
+	max  = 10
+	play = 1
+)
+
+type Movie struct {
+	Length  int
+	Name    string
+	viewers int
+	plays   int
+	ratings []float32
+}
+
+type Theatre struct {
+	name string
+}
+
+func (m *Movie) Rate(rating float32) error {
+	if 0 == m.plays {
```

Comment:
In Go `if` statements should read: `if <actual> == <expected>`. Your `if` statements are backward and non-idiomatic.

```suggestion
	if m.plays == 0 {
```